webapp/nextjs/app/editor/_components/Preview/Preview.tsx

```
@@ -0,0 +1,76 @@
+          <button className={styles.closeButton} onClick={onClose}>
+            &times;
+          </button>
+        </div>
+        <div className={styles.content}>
+          <h1 className={styles.previewTitle}>{title || '（タイトル未入力）'}</h1>
+          <div className={styles.meta}>
+            <span>{today}</span>
+          </div>
+          <div
+            className={styles.body}
+            dangerouslySetInnerHTML={{ __html: html }}
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)